app/validators/form_password_validator.rb

```diff
@@ -40,13 +32,6 @@ def password_graphemes_length
     end
   end
 
-  def password_score
-    @password_score = ZXCVBN_TESTER.test(
```

Comment:
this is the only usage of `ZXCVBN_TESTER` so we can remove it, right? and likely remove `zxcvbn` from the Gemfile?